src/main/java/org/sonatype/repository/chef/internal/ChefRecipeSupport.groovy

```diff
@@ -105,4 +114,89 @@ abstract class ChefRecipeSupport
   protected ChefRecipeSupport(final Type type, final Format format) {
     super(type, format)
   }
+
+  /**
+   * Matcher for cookbook download.
+   */
+  static Matcher downloadMatcher() {
+    LogicMatchers.and(
+        new ActionMatcher(GET, HEAD),
+        new TokenMatcher('/api/v1/cookbooks/{cookbook:.+}/versions/{version:.+}/download'),
+        new Matcher() {
+          @Override
+          boolean matches(final Context context) {
+            context.attributes.set(AssetKind.class, AssetKind.COOKBOOK)
+            return true
+          }
+        }
+    )
+  }
+
+  /**
+   * Matcher for cookbook details.
+   */
+  static Matcher cookbookDetailVersionMatcher() {
+    LogicMatchers.and(
+        new ActionMatcher(GET, HEAD),
+        new TokenMatcher('/api/v1/cookbooks/{cookbook:.+}/versions/{version:.+}'),
+        new Matcher() {
+          @Override
+          boolean matches(final Context context) {
+            context.attributes.set(AssetKind.class, AssetKind.COOKBOOK_DETAIL_VERSION)
+            return true
+          }
+        }
+    )
+  }
+
+  /**
+   * Matcher for cookbook details.
+   */
+  static Matcher cookbookDetailsMatcher() {
```

Comment:
Minor: This is really pedantic but it would helpful if these matchers were listed in operation order, i.e. I'm guessing it gets the cookbooks before the details before the version details and then does a download?